DigitalExcellence/dex-backend
# Feature/271-delete-redundant-files

## Description

Currently, there will be files in the filesystem and database that aren't linked to any project. The changes I made should fix this.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Create a project with a project icon
2. Delete/ update the project with another icon
3. The old icon should be deleted from the database

#271 

Closes: #271
